stl/inc/yvals_core.h

```diff
@@ -816,7 +833,7 @@
     4180 4324 4412 4455 4494 4514 4574 4582 4583 4587 \
     4588 4619 4623 4625 4626 4643 4648 4702 4793 4820 \
     4868 4988 5026 5027 5045 5220 5246 5278 5280 5281 \
-    6294                                              \
+    5285 6294                                         \

```

Comment:
I wonder whether global suppression is desired. Is it infeasible to separately suppress diagnostics for each templates?